flintrock/ec2.py

```diff
@@ -1014,6 +1014,24 @@ def get_clusters(*, cluster_names: list=[], region: str, vpc_id: str) -> list:
     return clusters
 
 
+def validate_tags(ctx, param, value):
+    """
+    Validate and parse optional ec2 tags as supplied on command line.
```

Comment:
I would just phrase this as "Validate and parse optional EC2 tags." since they can be specified via the config file too.